forge/forge/agent/base.py

```diff
@@ -274,16 +271,21 @@ async def run_pipeline(
                 raise e
         return method_result
 
-    def serialize_configs(self) -> str:
+    class ModelContainer(BaseModel):
+        models: dict[str, BaseModel]
```

Comment:
This is a utility class, shouldn't be part of `BaseAgent` imo

Reply:
I moved it outside the class and added `#HACK`. This is hopefully only temporary; once we update to pydantic v2 I'll refactor it (and `ModelContainer` will be unnecessary)